### ISSUE TITLE
Tileset LOD parent and child render at the same time

### ISSUE THREAD

**Issue by riematrix** [AnalyticalGraphicsInc/cesium#7007]
Hi, 
I followed the tileset example and fall into this problem. No matter how do I set the SSE or bounding volume, the refinement act strangely that either parent node and child node render at the same time or they don't show at all. 
My data contains 2 level tiles. A single parent and it's children. 

Data ### https://drive.google.com/open?id=1wdHxCpFgjPZMMjngMgLDUSRJoisrutDT

code
[3D Tiles Photogrammetry szyl.html.zip](https://github.com/AnalyticalGraphicsInc/cesium/files/2355327/3D.Tiles.Photogrammetry.szyl.html.zip)

PS, is there any tile set generator for converting existing LOD .obj tiles to cesium tilesets?

**Comment by likangning93** [comment]
Hi @riematrix, your `tileset.json` file specifies `ADD` in the `refine` field, when it seems like what you want is `REPLACE`. Take a look at the specification here: https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#refinement

The spec and `3d-tiiles-tools` projects are also a place to start for converting existing obj tiles to tilesets, but [Cesium ion](https://cesium.com/index.html) is also going to have the ability to tile large photogrammetry models.

**Comment by a0026** [comment]
The same problem !

**Comment by mramato** [comment]
@riematrix and @a0026 does setting `tileset.skipLevelOfDetails = false` make the problem go away?

@likangning93 @lilleyse could this be related to #7011 (or perhaps another bug introduced in the recent traversal changes?)

**Comment by likangning93** [comment]
@riematrix sorry for closing prematurely! I took a deeper look, and I noticed that the tileset provided isn't spec compliant. According to the current 3D Tiles specification, when a tile points to an external tileset it cannot have children. See here: https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#external-tilesets

You may have to refactor your tileset transforms a bit. Remember that transformations in a 3D Tileset are chained, so it may be easiest to have one transformation that puts the tileset where it needs to be on the Earth in the root and have either very small or nonexistent transformations for each tile.

@a0026 please let us know if you have the same problem in https://github.com/AnalyticalGraphicsInc/cesium/issues/7026.

**Comment by likangning93** [comment]
@lilleyse since it seems like this might be something that users encounter frequently, should Cesium check and throw some kind of exception for this case?

**Comment by lilleyse** [comment]
@likangning93 Yeah it seems like that is the problem. The root tile is refining to both the level 5 tile and the level 6 tiles simultaneously. The problem occurs in previous version of Cesium too (tried 1.48 and 1.45).

[Debugging demo I've been using for testing](http://localhost:8080/Apps/Sandcastle/index.html#c=XVHJasMwEP0V4YtdCJJDoBTXCaVpb4UeHHLSRZEntlotRovTpPTfK8cOTaqLmHmL5o16ZlEv4AAWLZGGA1qDE0Hh7bmXpfxcro32TGiw6Qx9U43iaaTZASrQnkkHVP/cPVJNdR/tvJDgwN/6jdfiZTOC2eQSrCxQ2nrfFYRIw5lsjfPFQ54vxnLiO9Ic9/dkssYfzuh0NlrUsAtN1ZrDswm6FrrZGhnUMJq3Aa5JayONFSc4e97iin0JFVTFLYCuOsbh1VpjI2me5/lfvPNWsOOgAXdWKOFFDw6zus6m2QbiRDsZozbmGqA6mSWl80cJq/HhJ6E6Y/2wiAxj4kF1knlwZBf4Z0zKnRuEJbmIylr0SNRLmvz7GpogLplzEdkHKasYlCarkkT+jUwaNmzpvQcr2XGgtPPV29jEGJcklhfVLw)

**Comment by lilleyse** [comment]
> @lilleyse since it seems like this might be something that users encounter frequently, should Cesium check and throw some kind of exception for this case?

Hard to say, I haven't seen this problem until now. We try not to do too much validation and prefer to offload that to the 3D Tiles validator, well once it's ready.

**Comment by mramato** [comment]
I would strongly suggest that anything that is simple to check for, is a fairly common mistake, and does not affect performance is something we spit out a console warning for. (unless unrecoverable, then a RuntimeError).

Not everyone is going to run the validator and not everyone is even in control of generating the tileset.  It will create a lot of noise in our issue tracker and the forum if we have to continual help people debug their tilesets for things that Cesium could have easily checked for.

That being said, it sounds like this might not be a common mistake, so maybe just put it on the radar and see if it pops up again?

**Comment by a0026** [comment]
> @riematrix sorry for closing prematurely! I took a deeper look, and I noticed that the tileset provided isn't spec compliant. According to the current 3D Tiles specification, when a tile points to an external tileset it cannot have children. See here: https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#external-tilesets
> 
> You may have to refactor your tileset transforms a bit. Remember that transformations in a 3D Tileset are chained, so it may be easiest to have one transformation that puts the tileset where it needs to be on the Earth in the root and have either very small or nonexistent transformations for each tile.
> 
> @a0026 please let us know if you have the same problem in #7026.

Hello. I do have the same problem in #7026, and I will try as you mentioned before. Thanks!
